docs/faq/README.md

```diff
@@ -78,3 +78,19 @@ If either symbol fails to display correctly, your system is still misconfigured.
 Unfortunately, getting font configuration correct is sometimes difficult. Users
 on the Discord may be able to help. If both symbols display correctly, but
 you still don't see them in starship, [file a bug report!](https://github.com/starship/starship/issues/new/choose)
+
+## How do I uninstall Starship?
+
+Starship is just as easy to uninstall as it is to install in the first place.
+
+1. Remove any lines in your shell config (e.g. `~/.bashrc`) used to initialize Starship.
+1. Delete the Starship binary.
```

Comment:
This renders correctly in the preview. Is it idiomatic to use the same number in markdown lists and just let the renderer sort out the actual numbering?

Reply:
It comes down to personal preference. I've run into the problem of removing a step and being left with incorrect numbering one too many times. I wouldn't mind changing it though.

Reply:
Nah, I just didn't realize markdown could do this for you. Might start doing that myself in the future--I've had similar issues with numbering not updating.